src/TagLib/Riff/AviHeaderList.cs

```diff
@@ -99,12 +99,17 @@ public class AviHeaderList
 					"Invalid header length.");
 			
 			header = new AviHeader (header_data, 0);
-			
-			foreach (ByteVector list_data in list ["LIST"])
-				if (list_data.StartsWith ("strl"))
-					codecs.Add (AviStream
-						.ParseStreamList (list_data)
-						.Codec);
+
+			foreach (ByteVector list_data in list["LIST"])
+			{
+				if (list_data.StartsWith("strl"))
+				{
+					AviStream stream = AviStream
+						.ParseStreamList(list_data);
```

Comment:
Let's just put this on one line now that there is only one `.`

Reply:
done !